frontend/public/components/storage-class-form.tsx

```diff
@@ -681,9 +693,15 @@ export class StorageClassForm_ extends React.Component<StorageClassFormProps, St
   };
 
   getProvisionerElements = () => {
-    const dynamicContent = Object.keys(this.storageTypes[this.state.newStorageClass.type].parameters).map(key => {
-      const parameter = this.storageTypes[this.state.newStorageClass.type].parameters[key];
-      const validationMsg = _.get(this.state.newStorageClass.parameters[key], 'validationMsg', null);
+    const parameters = this.storageTypes[this.state.newStorageClass.type].parameters;
+
+    if (_.isEmpty(parameters)) {
+      return null;
+    }
+
+    const dynamicContent = Object.keys(parameters).map(key => {
```

Comment:
Existing code, but this is better as `_.map`

```suggestion
    const dynamicContent = _.map(parameters, (parameter, key) => {
```